queries/sql/rainbow-delimiters.scm

```diff
@@ -1,24 +1,24 @@
 (subquery
-    "(" @opening
-    ")" @closing) @container
+    "(" @delimiter
+    ")" @delimiter @sentinel) @container
 
 (invocation
-    "(" @opening
-    ")" @closing) @container
+    "(" @delimiter
+    ")" @delimiter @sentinel) @container
 
 (list
-    "(" @opening
-    ")" @closing) @container
+    "(" @delimiter
+    ")" @delimiter @sentinel) @container
 
 ;; A parenthesized expression; the delimiters are actually outside the
 ;; expression, but that's OK because the nesting level only looks at containers
 (
-    "(" @opening
+    "(" @delimiter
     .
     [
         (binary_expression)
         (literal)
     ] @container
```

Comment:
```suggestion
    ]
```
This `@container` should be moved to the end. But even with that fix, sql highlighting seems pretty broken in the example file.

Reply:
I added a fix to this now. It works perfectly for me as long as you don't have more than 3 layers of (((()))). I.e., ((())) seems to be ok.